corehq/apps/smsbillables/static/smsbillables/js/smsbillables.rate_calc.js

```diff
@@ -130,3 +130,26 @@ var Select2SmsRateHandler = function (options) {
 
 Select2SmsRateHandler.prototype = Object.create( BaseSelect2Handler.prototype );
 Select2SmsRateHandler.prototype.constructor = Select2SmsRateHandler;
+
+$(function() {
+    _.each($(".ko-sms-rate-calculator"), function(element) {
```

Comment:
I think I just prefer to use jQuery's `each` when I've got a jQuery collection because it's less !@#$%^&*_ characters, and therefore marginally more readable :)